src/routes/positions/positions.service.spec.ts

```diff
@@ -161,13 +161,20 @@ describe('PositionsService', () => {
     const aave = res.find((p) => p.protocol === 'Aave')!;
     const usdcGroup = aave.items.find((g) => g.name === 'USDC')!;
 
-    expect(usdcGroup.items).toHaveLength(1);
-    expect(usdcGroup.items[0]).toEqual(
-      expect.objectContaining({
-        position_type: PositionType.deposit,
-        balance: '15',
-        fiatBalance: '45',
-      }),
+    expect(usdcGroup.items).toHaveLength(2);
+    expect(usdcGroup.items).toEqual(
+      expect.arrayContaining([
+        expect.objectContaining({
+          position_type: PositionType.deposit,
+          balance: '10',
+          fiatBalance: '30',
+        }),
+        expect.objectContaining({
+          position_type: PositionType.deposit,
+          balance: '5',
+          fiatBalance: '15',
+        }),
```

Comment:
On Zerion, multiple positions of the same type are aggregated. Why don't we want to do this anymore?